posthog/management/commands/create_bulk_events.py

```diff
@@ -0,0 +1,125 @@
+                                                    'delete' for deleting bulk demo data 
+                                                    or 'create' for creating bulk demo data;
+                                                    default 'create'
+                                                    eg. --mode delete
+                                                     """
+                            )
+                            
+        parser.add_argument('--count', nargs='+', default=['1M'], help="""
+                                                        '1M' for creating about 1 million random events
+                                                        or 
+                                                        '100K' for creating about 100 thousand random events
+                                                        default 1M
+                                                        """
+                            )
+
+    def handle(self, *args, **options):
+        team_id = options['team_id']
+        mode =  options['mode'][0]
+        count = options['count'][0]
+
+        if not team_id:
+            print ("Please specify the --team id")
+            return
+
+        team = Team.objects.get(pk=team_id[0])
+
+        if count.lower() == "100k":
+            limit = 1000
+        else:
+            limit = 10000
+            print("This may take a while...")
+
+
+        with open(Path('posthog/demo_data.json').resolve(), 'r') as demo_data_file:
+            demo_data = json.load(demo_data_file)
+        
+        base_url = '127.0.0.1/bulk_demo/'
+
+        if mode.lower() == 'delete':
+            self._delete_demo_data(team)
+        else:
+            self._delete_demo_data(team)
+            self._create_funnel(base_url, team)
+            self._create_events(demo_data,team, limit)
+        
+    def _create_events(self, demo_data, team, limit):
+        Person.objects.bulk_create([
+        Person(team=team, properties={'is_demo': True}) for _ in range(0, 100)])
+
+        distinct_ids: List[PersonDistinctId] = []
+        events: List[Event] = []
+        days_ago = 7
+        demo_data_index = 0
+ 
+        base_url = '127.0.0.1/bulk_demo/'
+
+        for index, person in enumerate(Person.objects.filter(team=team)):
+            print (index)
+            if index > 0 and index % 14 == 0:
+                days_ago -= 1
+
+            distinct_id = str(uuid.uuid4())
+            distinct_ids.append(PersonDistinctId(team=team, person=person, distinct_id=distinct_id))
+            date = now() - relativedelta(days=days_ago)
+
+            if index % 3 == 0:
+                person.properties.update(demo_data[demo_data_index])
+                person.save()
+                demo_data_index += 1
+
+                for i in range(3):
+                    Event.objects.bulk_create([Event(
+                        event=random.choice(['$autocapture', '$pageview', '$signup']), team=team, distinct_id=distinct_id, 
```

Comment:
could you re-format this so that each property appears on one line, right now this looks pretty unreadable

and the dict inside properties I guess too